src/Database/Mysql/Driver/Mysql.php

```diff
@@ -39,6 +40,10 @@ public function create(Tenant $tenant): bool
     {
         $config = $this->configure($tenant);
 
+        if (isset($config['allowedHost'])) {
+            $config['host'] = $config['allowedHost'];
+        }
```

Comment:
1. Why aren't these lines part of configure()?
2. What is a `allowedHost` that word makes no sense to me.

Reply:
1. It would cause issues when creating a normal tenant connection. When a normal tenant connection is configuring, we would not want the host to be % as it would result in errors. We could make a simple additional function that does this instead of repeating it 3 times.
2. I wish I knew a better word, really open to suggestions on this one xD